config/build-chartcuterie.ts

```diff
@@ -77,6 +77,21 @@ async function runEsbuild(commitHash: string): Promise<void> {
     minify: false,
     treeShaking: true,
     logLevel: 'info',
+    loader: {
+      '.svg': 'file',
```

Comment:
It looks as though we were getting lucky that the loaders existed before because of tree-shaking, we may not need these in charcuterie, but without them it'll fail to build